utils/README.md

```diff
@@ -30,17 +41,17 @@ Copying your ssh key to the linux server allows you to connect to the server fro
 
 1. Generate ssh key (if not already generated):
 
-If you dont already have a ssh key generated on you machine, run:
+   If you dont already have a ssh key generated on you machine, run:
 
-```
-ssh-keygen -t rsa -b 4096
-```
+   ```
+   ssh-keygen -t rsa -b 4096
+   ```
 
 2. Copy key to your linux server
    - change `{IP}` to the IP or domain of your server and `{username}` to your created user
    - on Windows:
    ```
-   type $env:USERPROFILE\.ssh\id_rsa.pub | ssh {username}@{IP} "cat >> .ssh/authorized_keys"
+   type $env:USERPROFILE\.ssh\id_rsa.pub | ssh {username}@{IP} "mkdir -p ~/.ssh && cat >> ~/.ssh/authorized_keys"
```

Comment:
Only "mkdir" doesn't worked for me many times. Maybe add "touch ~/.ssh/authorized_keys" also.

Reply:
I added -p which fixes this issue, see [this stackoverflow question](https://stackoverflow.com/questions/793858/how-to-mkdir-only-if-a-directory-does-not-already-exist)